specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-09-01/managedClusters.json

```diff
@@ -2693,7 +2699,6 @@
           "description": "The current deployment or provisioning state."
         },
         "powerState": {
-          "readOnly": true,
           "description": "Describes whether the Agent Pool is Running or Stopped",
```

Comment:
Please update this description to explain in what situations you can include this field in a PUT and what it does if you do.